extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -443,8 +383,8 @@ public function categoryTopArticles() {
 				$namespace = (new Wikia\Search\MediaWikiService)->getNamespaceIdForString( $colonSploded[0] );
 				// remove "Category:", since it doesn't work with ArticlesApiController
 				$category = ( is_int( $namespace ) && $namespace == NS_CATEGORY )
-				         ? implode( ':', array_slice( $colonSploded, 1 ) )
-				         : $category;
+						 ? implode( ':', array_slice( $colonSploded, 1 ) )
+						 : $category;
 				//@todo use single API call here when expansion is released
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)